app/ui/window.js

```diff
@@ -216,7 +216,8 @@ module.exports = class Window {
       window.on(ev, () => rpc.emit('windowGeometry change'));
     }
     rpc.win.on('move', () => {
-      rpc.emit('move');
+      const position = window.getPosition();
+      rpc.emit('move', {position: {x: position[0], y: position[1]}});
```

Comment:
This is a common usage to not separate position and size.

I would trigger:
```
const position = window.getPosition();
rpc.emit('move', {bounds: {x: position[0], y: position[1]}});
```

Or directly:
```
const bounds = window.getBounds();
rpc.emit('move', {bounds});
```

Should we do the same when window size has changed?

Reply:
The reason i wrote `{x: position[0], y: position[1]}}` is because
```
const bounds = window.getBounds();
rpc.emit('move', {bounds});
```
render as  ex: `{0 : 24, 1: 52}` note that those are just random number as example.

Reply:
No, `window.getPosition()` returns an array (and you were right to decompose it), but `window.getBounds()` returns `{ x: 192, y: 147, width: 624, height: 469 }` for example.

Reply:
I didn't comment on your decomposition. Only on your variable name and your variable content ;)

Reply:
Yeah, indeed you are right. However, for the window move action doesn't invoke a window resize. So, no need to send unnecessary data to the reducer 😄

Reply:
Should be good to go 🚀

Reply:
Fixed: Updated win for window on `rpc.js`